CodeMaid/Logic/Reorganizing/CodeReorganizationManager.cs

```diff
@@ -364,9 +364,8 @@ private void RecursivelyReorganize(IEnumerable<BaseCodeItem> codeItems, ICodeIte
 
             // Get the items in their current order and their desired order.
             var currentOrder = GetReorganizableCodeItemElements(codeItems);
-            var desiredOrder = currentOrder.OrderBy(CodeItemTypeComparer.CalculateNumericRepresentation)
-                                           .ThenBy(x => Settings.Default.Reorganizing_AlphabetizeMembersOfTheSameGroup ? (object)x.Name : (object)x.StartOffset)
-                                           .ToList();
+            var desiredOrder = new List<BaseCodeItemElement>(currentOrder);
+            desiredOrder.Sort(new CodeItemTypeComparer(Settings.Default.Reorganizing_AlphabetizeMembersOfTheSameGroup));
```

Comment:
Cool, thank you for confirming.  Agree that StartOffset should break any ties, as even variables on the same line (e.g. `int x, y;`) have different character offsets.